net-dialup/radiusclient/files/radiusclient-0.3.2-BJA-cross.diff

```diff
@@ -0,0 +1,67 @@
+--- configure.in	2002-02-03 14:51:40.000000000 +0100
```

Comment:
The patch comes from me as metionned in the pach head. Patch was reported upstream on 28/06/2011 15:22 http://freecode.com/projects/radiusclient/tickets/2c46a43c4ab what now leads to a 404 error. Looking at sending it back